### PR TITLE
[INLONG-9873][Manager] Support adding data add tasks for file collection

### LINKED ISSUE #9873: [Improve][Manager] Support adding supplementary recording tasks for file collection
### Description

Support adding supplementary recording tasks for file collection.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #9873

### Motivation

Support adding data add tasks for file collection.

### Modifications

Support adding data add tasks for file collection.

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java [hunk 1]
```diff
@@ -629,4 +630,12 @@ private void chkUnmodifiableParams(SourceRequest request) {
         request.setInlongStreamId(entity.getInlongStreamId());
         request.setSourceName(entity.getSourceName());
     }
+
+    @Override
+    public Integer addSub(SubSourceRequest request, String operator) {
+        StreamSourceEntity entity = sourceMapper.selectById(request.getSourceId());
+        StreamSourceOperator sourceOperator = operatorFactory.getInstance(entity.getSourceType());
+        int id = sourceOperator.addSubSource(request, operator);
+        return id;
```
Comment (aloyszhang) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java:639]:
```suggestion
         return sourceOperator.addSubSource(request, operator);
```

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java [hunk 2]
```diff
@@ -190,6 +190,11 @@ void updateStatusByIds(@Param("idList") List<Integer> idList, @Param("status") I
      */
     void updateStatusByDeleted();
 
+    /**
+     * Logic delete the sub source by modifiy time
+     */
+    void logicalDeleteByTimeout(@Param("beforeDay") Integer beforeDays);
+
```
Comment (aloyszhang) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java:197]: Please clarify the meaning of parameter `beforeDays`

File: inlong-manager/manager-web/src/main/resources/application-dev.properties [hunk 3]
```diff
@@ -97,6 +97,11 @@ source.update.enabled=false
 source.update.before.seconds=60
 source.update.interval=60
 
+# If turned on, Regularly clear expired supplementary recording tasks
+subSource.cleansing.enabled=false
+subSource.cleansing.interval=10
```
Comment (aloyszhang) [inline inlong-manager/manager-web/src/main/resources/application-dev.properties:102]: It's better to include time unit in the configuration name.

File: inlong-manager/manager-web/src/main/resources/application-prod.properties [hunk 4]
```diff
@@ -96,6 +96,11 @@ source.update.enabled=false
 source.update.before.seconds=60
 source.update.interval=60
 
+# If turned on, Regularly clear expired supplementary recording tasks
+subSource.cleansing.enabled=false
+subSource.cleansing.interval=10
```
Comment (aloyszhang) [inline inlong-manager/manager-web/src/main/resources/application-prod.properties:101]: ditto

File: inlong-manager/manager-web/src/main/resources/application-test.properties [hunk 5]
```diff
@@ -97,6 +97,11 @@ source.update.enabled=false
 source.update.before.seconds=60
 source.update.interval=60
 
+# If turned on, Regularly clear expired supplementary recording tasks
+subSource.cleansing.enabled=false
+subSource.cleansing.interval=10
```
Comment (aloyszhang) [inline inlong-manager/manager-web/src/main/resources/application-test.properties:102]: ditto

Comment (fuweng11) [reply inlong-manager/manager-web/src/main/resources/application-test.properties]: Fixed.

File: inlong-manager/manager-web/src/main/resources/application-prod.properties (hunk 4 above)
Comment (fuweng11) [reply inlong-manager/manager-web/src/main/resources/application-prod.properties]: Fixed.

File: inlong-manager/manager-web/src/main/resources/application-dev.properties (hunk 3 above)
Comment (fuweng11) [reply inlong-manager/manager-web/src/main/resources/application-dev.properties]: Fixed.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java (hunk 2 above)
Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java]: Fixed.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/StreamSourceServiceImpl.java]: Fixed.

**Review by aloyszhang — DISMISSED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]
LGTM

**Pull request merged**